### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by pmoura** [LogtalkDotOrg/logtalk3#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PaulBrownMagic** [comment]
Nice! Happy to be involved.
I particularly like using Logtalk for reasoning about situations and actions, as well as ontological reasoning. I also use swilgt for making web applications.
If folks want to know more about me, I keep a [little blog](http://pbrown.me)

**Comment by jacintodavila** [comment]
Hello Paulo et al, I am newcomer to logtalk3 but not to logic programming. I have been working at https://logicalcontracts.com with a new form of controlled natural language: Logical English, proposed by B. Kowalski. I look forward to ways of building on top of logtalk too. Regards

**Reply by Stranger2015** [reply]
Hi Paulo and everyone,
I'm just deep in thoughts of Prolog/HiLog/Logtalk triple.

Regards
